website/blog/2023-09-06-optimize-try-cast.mdx

```diff
@@ -0,0 +1,114 @@
+---
+slug: optimize-try_cast.
+title: "Learnings from optimizing try_cast"
+authors: [lsakka]
+tags: [tech-blog, performance]
+---
+
+One of the queries shadowed internally at Meta was much slower in Velox compared to presto(2 days vs 4.5h). Initial investigation identified that the overhead is related to casting empty strings inside a try_cast.
+
+In this blogpost I summarize my learnings from investigating and optimizing try_cast.
+
+--------------------------------------------------------------------------------------------
+
+### Start and end results
+
+Initial benchmark:
```

Comment:
Its not clear what the values in the right column are. Is it Presto vs Prestissimo timings ? 

(Same comment for other timings in this writeup)

Reply:
I will add headers